prow/plugins/project/project.go

```diff
@@ -114,22 +114,42 @@ func updateProjectNameToIDMap(projects []github.Project) {
 // proposed column name, whether the command is to remove issue/PR from project,
 // and the error message
 func processRegexMatches(matches []string) (string, string, bool, string) {
-	var shouldClear = false
-	proposedProject := matches[1]
+	proposedProject := ""
 	proposedColumnName := ""
-	if len(matches) > 1 && proposedProject != clearKeyword {
-		proposedColumnName = matches[2]
+
+	var shouldClear = false
+	content := strings.TrimSpace(matches[1])
+
+	// Take care of clear
+	if strings.HasPrefix(content, clearKeyword) {
```

Comment:
Can we have the prefix as `clear ` instead (also same for replace)? The `parts = strings.SplitN(content, " ", 2)` down below might be problematic because of this. For a command like `/project clear project name`, the `parts` will be 
`part[0]`: `(empty string)`
`part[1]`: `project name`

Reply:
I have tried that in the first place `clear ` failed a few test cases, I think it's supporting `/project clear` as a valid command. The problem you mentioned did happen, trimming spaces at both ends solved the problem, it's tested by the test case you suggested below

Reply:
Ah I see. Sounds good!